docs/segment.md

```diff
@@ -2,98 +2,188 @@
 
-- ++ctrl++ and drag allows to move all the selected control points at once (or the selected lines if no control points are selected).
+![image: Drawing a baseline freehand.](img/segment/baseline_freehand.gif "You can freehand draw baselines on eScriptorium, however it is not recommended.")
 
-- The red trash button deletes all selected lines/regions,
-- The yellow trash button only deletes selected control points.
+- We recommend to segment text lines by clicking on the beginning of your text lines, and then clicking where you want your text line to end. The text line will be straight.
 
-<!--## Editing lines
+![image: Drawing a straight baseline.](img/segment/baseline_straight.gif "You can draw straight baselines.")
 
-## Editing regions -->
+The mask of the segmented text line is then automatically calculated by eScriptorium by clicking on the **thumbs up** button. You can display three views of the baselines with the **toggle line masks and stroke width** button, or by pressing ++m++. One of them will let the mask appear.
 
-## Reordering
+eScriptorium also calculate the mask associated with each line asynchronously, you do not need to always press the dedicated button.
 
-The playback order of the segments is calculated automatically. You can display the order number of each segment from the “Segmentation” panel by clicking on “Toggle ordering display (L)” or from the “Text” panel where the lines are displayed in the following order.
+You can delete a baseline by selecting it, and by clicking on the thrash button, or by pressing ++delete++.
 
-It is possible to modify this order from the “Text” panel by clicking “Toggle sorting mode.” A simple drag and drop of the lines is then enough to carry out the modification.
+You can change the reading order of a baseline by selecting it, and then clicking on the **reverse selected lines** button or by pressing ++i++.
+
+![image: Calculating the masks.](img/segment/calculate_masks.gif "You can manually activate the mask calculation by pressing the calculate masks button.")
+
+!!! tip
+    You can select several text lines by pressing ++shift++ and by click and drag around them.
+
+!!! note
+    Keep in mind that you can always press ++ctrl+z++ to undo any modification.
+
+### Labeling segmented text lines with an ontology
+
+eScriptorium allows to label segmented text lines with an ontology that you can customize. **This process is exactly the same as the one we described earlier for text regions.**
+
+The ontology panel is one of the tabs you can access once you are working in a document. It is accessible with (`{base_url}/document/{document-id}/ontology/`). Text lines have a dedicated field named *line types*.
+
+eScriptorium uses four line types by default:
+
+- Correction
+- Main
+- Numbering
+- Signature
+
+You can use them, or create your own. To deactivate default line types, you can deactivate them by unticking the ones you want to discard, and then by clicking on the update button.
```

Comment:
Same comment as for the text regions

Reply:
To add to this: I think there should be only one section explaining how the ontology works. They basically say the same info, and it will be easier to maintain later.